### DISCUSSION TITLE
plugin load template in settings view

### DISCUSSION THREAD

**Discussion by wolflu05** [inventree/InvenTree#4071] [Q&A] [answered]
Hello,

I'm continuing working on my bulk creation plugin. I already made good progress. Currently I have my settings load as a custom panel on the StockIndex view. But is there an option for my plugin to add some custom html and js over here like categories and parts have? 
![image](https://user-images.githubusercontent.com/76838159/208187189-64396235-f727-46f7-9282-a7e00f89c3e8.png)

I know that there is a settings mixin which already adds some text fields to this page, but my settings are way more complex than just simple text fields or booleans.

**Comment by SchrodingersGat** [comment, marked as answer]
There is no way currently to add completely custom information to the plugin settings page. It would make sense to add this as a feature, at the very least each plugin could (optionally) define a custom rendering method for its own plugin configuration tab.

**Reply by wolflu05** [reply]
I created #4073
